### ISSUE TITLE
Namespaced remotes created incorrectly

### ISSUE THREAD

**Issue by safazi** [roblox-aurora/rbx-net#67]
Version: 3.0.0

Code:
```ts
import Net, { Definitions as def } from "@rbxts/net";

export = Net.CreateDefinitions({
    initialize: def.ServerAsyncFunction<() => boolean>(),

    realm: def.Namespace({
        subscribe: def.ServerToClientEvent<[realm: string]>(),
        unsubscribe: def.ServerToClientEvent<[realm: string]>(),
    }),
});
```

Produces:
![image](https://user-images.githubusercontent.com/16279574/162847916-76b3a2fe-c42d-4fdb-a5f1-df2ba4f1c612.png)

**Comment by safazi** [comment]
Worth noting that calling
```ts
definitions.Server.GetNamespace("realm").Get("subscribe");
```
will create the remote properly

![image](https://user-images.githubusercontent.com/16279574/162848250-672a2d12-79da-4c94-b4ae-056a09d41526.png)

**Comment by Vorlias** [comment]
Sorry this took so long, should be fixed in 3.0.2.
